test/sdf_gtest.cpp

```diff
@@ -6,7 +6,7 @@
 
 #include "tue/filesystem/path.h"
 
-tue::config::ReaderWriter config;
+static tue::config::ReaderWriter config;
```

Comment:
I would expect the store to be setup as part of each test, instead of being shared, as now tests may influence each other leading to false test results.

Reply:
I reverted the addition of the static. I don't know anymore why I did it.

The tests do 'depend' on the previous tests. At least on bringing the object to a correct state. It isn't the cleanest way. If this is really wrong, I will create an issue, so this can be fixed in an other PR. So let me know.

Reply:
@alberth please see my previous comment.